bandwidth-file-spec.txt

```diff
@@ -208,6 +208,82 @@ It consists of:
 
     This Line has been added in version 1.1.0 of this specification.
 
+  "percent_eligible_relays=" Int NL
+
+    [Zero or one time.]
+
+    The number of eligible relays, as a percentage of the number
+    of relays in the consensus.
+    
+    This line SHOULD be equal to:
+        (number_eligible_relays * 100.0) / number_consensus_relays
+    to the number of relays in the consensus to include in this file.
+
+    Implementations SHOULD include this KeyValue when
+    "minimum_percent_eligible_relays" is minor than
+    "percent_eligible_relays" and MAY not include any RelayLine.
+
+    This Line has been added in version 1.2.0 of this specification.
```

Comment:
Please add the new format version 1.2.0 in section 1.4:
https://github.com/torproject/torspec/blob/master/bandwidth-file-spec.txt#L43